pkg/controller.v1/mxnet/mxjob_controller.go

```diff
@@ -448,17 +456,12 @@ func onOwnerCreateFunc() func(event.CreateEvent) bool {
 		}
 
 		// TODO: check default setting
```

Comment:
Can we remove the TODO?

Reply:
I doubt check this comment. I should change the description. Let's keep it for now. 

I remember the reason I put a TODO there is because we use `mxjobv1.SetDefaults_MXJob(mxjob)` to set defaults, We can use `scheme.Scheme.Default(mxjob)`,  This is equivalent actually  but I prefer to change to `scheme.Scheme.Default(mxjob)` in the near future. 

Currently, we have not registered defaulters in scheme so that `scheme.Scheme.Default(mxjob)` do nothing there.

Reply:
/cc @gaocegege